bin/docker/run

```diff
@@ -6,7 +6,7 @@
 set -e
 
 # Environment variables to export (Travis-defined + BEAVER_DOCKER_VARS)
-env_vars="$(printenv -0 | sed -zn 's/^\([^=]\+\)=.*$/\1\n/p')"
+env_vars="$(printenv -0 | sed -zn 's/^\([^=]\+\)=.*$/\1\n/p' | tr -d '\0')"
```

Comment:
Mmm, strange, I never saw any warnings... But then I don't use `bash` as `sh`, so maybe that's it.